### PR TITLE
logo add

### PR DESCRIPTION
**Pull request by psikoz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# place-cursor-at
+<p align="left"><img src="logo/logotype_horizontal.png" alt="place-cursor-at" height="100px"></p>
```
Comment (unclechu) [inline README.md:1]: 1. I don't get what `<p ...>` here for? Maybe remove it?
2. I believe setting `height` as `Npx` is invalid HTML, you supposed to remove `px` suffix.

File: logo/icon.svg [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="153.144px" height="215.509px" viewBox="0 0 153.144 215.509" enable-background="new 0 0 153.144 215.509"
```
Comment (unclechu) [inline logo/icon.svg:5]: It would be better to get rid of floating point in document size.

File: logo/logotype_horizontal.svg [hunk 3]
```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="912.237px" height="174.961px" viewBox="0 0 912.237 174.961" enable-background="new 0 0 912.237 174.961"
```
Comment (unclechu) [inline logo/logotype_horizontal.svg:5]: It would be better to get rid of floating point in document size.

File: logo/logotype_vertical.svg [hunk 4]
```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="424.741px" height="232.74px" viewBox="0 0 424.741 232.74" enable-background="new 0 0 424.741 232.74"
```
Comment (unclechu) [inline logo/logotype_vertical.svg:5]: It would be better to get rid of floating point in document size.

**Review by unclechu — CHANGES_REQUESTED** [review]
We also must agree about some license to distribute this artwork. Something from the Creative Commons. I'd say: CC BY SA (see https://en.wikipedia.org/wiki/Creative_Commons_license), please say explicitly if you agree with this decision.

**Comment by psikoz** [comment]
This logo is licensed under a Creative Commons Attribution 4.0 International License. (https://creativecommons.org/licenses/by/4.0/)

@unclechu , I don't understand very well from github. so I don't want to ruin your repository :/

I have uploaded all the files to the googledrive. Could you please make these changes?

https://drive.google.com/open?id=1wJrVknT4qS27T4eLTyBvimwM6Goal3LM

**Comment by unclechu** [comment]
@psikoz NP, I'll make these changes by myself. Anyway, you wouldn't mess anything because this PR made on your side, your fork (copy of this repo). Any new commit to your fork to proper branch will be added to this PR.

**Comment by psikoz** [comment]
Okay, thank you for it.

**Comment by unclechu** [comment]
@psikoz I pulled your branch and did some minor adjustments. Now the change is merged. I’m closing this pull request as accepted and merged. Thanks for you contribution!

**Pull request closed without merging**